RazorHtmlEmails.RazorClassLib/Views/RegistrationConfirmation.cshtml.cs

```diff
@@ -0,0 +1,18 @@
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace RazorHtmlEmails.RazorClassLib.Views
+{
+    public class RegistrationConfirmation : PageModel
+    {
+        private string UserName;
```

Comment:
fields = _userName
properties = UserName

Reply:
Yeah, in should be a property. Good call, thanks.